scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -100,7 +104,10 @@
 import com.google.common.collect.ImmutableSet;
 import com.google.common.collect.Iterables;
 
+import it.sauronsoftware.cron4j.Scheduler;
```

Comment:
You use an external dependency cron Scheduler. Doesn't our Scheduler have the ability to do the same?